docs/index.rst

```diff
@@ -23,6 +23,24 @@ sunkit-image
           generated/gallery/index
           changelog
 
+Mission Statement
+=================
+
+The goal of the ``sunkit-image`` library is to provide access to image processing routines that are:
+
+1. Focused on being applied to solar physics data.
```

Comment:
```suggestion
1. Focused on being applied to solar image data.
```

We need to say something about what kind of date this package is focused on. It's specifically focused on operations on image data.

Reply:
I'm not sure if we want to more strictly define "image" here